transmart-schemas/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'org.springframework.boot' version '2.1.4.RELEASE'
+    id 'java'
+}
+
+apply plugin: 'io.spring.dependency-management'
+
+ext {
+    liquibaseVersion = '3.6.1'
+}
+
+dependencies {
+    compile 'org.springframework.boot:spring-boot-starter'
+    compile "org.liquibase:liquibase-core:${liquibaseVersion}"
+    compile "org.postgresql:postgresql:${postgresqlVersion}"
```

Comment:
where should the `${postgresqlVersion}` be configured?

Reply:
That is configured in the root [build.gradle](build.gradle)